service/handlers.go

```
@@ -407,6 +407,13 @@ func (h handler) getTarget(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	targetExists, _ := cp.TargetExists(projectName, targetName)
```

I think we should handle the error here instead of swallowing it.